ci/scripts/run_tests.sh

```diff
@@ -4,13 +4,7 @@ set -o xtrace
 
 BENCHMARK="${BENCHMARK:-false}"
 
-# Ensure we run additional tests when testing the latest coiled-runtime
-if [[ $COILED_RUNTIME_VERSION = 'latest' || $COILED_RUNTIME_VERSION = 'upstream' ]]
-then
-  EXTRA_OPTIONS="--run-latest"
-else
-  EXTRA_OPTIONS=""
-fi
+EXTRA_OPTIONS="--run-latest"
```

Comment:
The whole point of it was to have the test suite run on very obsolete versions of dask. It feels redundant now and I think it should be removed completely.